### ISSUE TITLE
Min-Max-Close button action consistency when used on hidden toolbar.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Javier-DarthPalpatine/firefox-hide-caption-titlebar-plus#255]
```
What steps will reproduce the problem?
1. Place min-max-close buttons on the menu bar.
2. Hide the menu bar
3. Press alt, and then click one of the min-max-close buttons.

What is the expected output?
The min-max-close buttons respond before the toolbar hides itself.

What do you see instead?
The close button works - that's good.
But the min / max buttons don't.

What version of "Hide Caption Titlebar Plus" addon are you using?
2.8.6

What version of Firefox?
30.0

On what operating system?
Windows 8.1

- Please provide any additional information below.

From your changelog:
[Custom Close-button] Improvement in response to quick user clicks. You should 
successfully close your tabs quicker now (Ie. 'throwing' the mouse at top-right 
screen corner & pressing primary button). In next version, You'll be able to 
use *secondary* mouse button too & do more with this close button :-) 

Basically, make this change to the min and max buttons as well. : )
```

Original issue reported on code.google.com by `PimpUigi` on 11 Jun 2014 at 1:25
